### PR TITLE
FOGL-2558: Fix Debian packaging

### PR DESCRIPTION
**Pull request by pintomax**
FOGL-2558: Fix Debian packaging

### CODE REVIEW

File: make_deb [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    echo "export FOGLAMP_ROOT=<basedir>"
+    exit 1
+fi
+
+# Check/set LD_LIBRARY_PATH
+libPathSet=0
+libdir=${FOGLAMP_ROOT}/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1
+libdir=${FOGLAMP_ROOT}/cmake_build/C/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1
+[ "$libPathSet" -eq "0" ] && echo "Unable to set/update LD_LIBRARY_PATH to include path of Foglamp shared libraries: check whether ${FOGLAMP_ROOT}/lib or ${FOGLAMP_ROOT}/cmake_build/C/lib exists" && exit 1
+
```
Comment (praveen-garg) [inline make_deb:119]: Call the requirements.sh here

**Review by praveen-garg — CHANGES_REQUESTED** [review]
Update readme with dependency requirements info (and script call)

File: make_deb (hunk 1 above)
Comment (pintomax) [reply make_deb:119]: @praveen-garg requirements.sh, with libcurl check, as in other plugins, is not needed here

Comment (praveen-garg) [reply make_deb:119]: ah! yes, its a rule plugin only.

Please remove the requirements.sh from this branch as well.

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**